packages/tools/kadena-cli/package.json

```diff
@@ -84,6 +84,6 @@
     "eslint": "^8.45.0",
     "prettier": "~3.0.3",
     "typescript": "5.2.2",
-    "vitest": "^0.34.6"
+    "vitest": "^1.1.0"
```

Comment:
Should it be updated in all other packages as well?

Reply:
all other packages are already `1.1.0` only cli package was behind because of feature repo. But, I think this is already updated. I could see some merge conflicts. I'll rebase and check it once